modules/policy/pages/index.adoc

```diff
@@ -199,7 +195,7 @@ $ kubectl get pods --all-namespaces --selector=app=skupper
 . Install the CRD as described in xref:installing-crd[].
 This step closes connections and removes all {service-network} services and exposed resources.
 
-. If Skupper sites exist in the cluster not created by `cluster-admin`, you must grant permissions to read policies to developers to avoid that site being blocked from the {service-network}.
+. If Skupper sites that were not created by `cluster-admin` exist in the cluster, you must grant permissions to read Skupper policies to developers to avoid that site being blocked from the {service-network}.
```

Comment:
The 'to's in close proximity make this a bit difficult to read.  

Maybe add a fullstop after developers and have a separate sentence "This avoids the site being blocked from...."

or change to following which might be a little easier to read:

If Skupper sites that were not created by `cluster-admin` exist in the cluster, you must grant developers permissions to read Skupper policies  to avoid that site being blocked from the {service-network}.